@here/harp-geometry/lib/SubdivisionModifier.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    Face3,
+    Geometry,
+} from "three";
+
+/**
+ * The `SubdivisionModifier` subdivides triangle mesh geometries positioned
+ * on the surface of a sphere centered at `(0, 0, 0)`.
+ */
+export class SubdivisionModifier {
```

Comment:
Maybe rename to SphereSubdivisionModifier b/c for terrain we might use a different approach.

Reply:
I renamed it to `SphericalGeometrySubdivisionModifier`.